Bottom: 0e405a56c9fcc559c734798fad86b5fdfa3269a4
Top:    281150822c899ebd6f86fdec7d0d756b53ba5672
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-12-27 18:12:22 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
